bokehjs/src/coffee/callback/customjs.coffee

```diff
@@ -1,5 +1,5 @@
 _ = require "underscore"
-Model = require "../models/model"
+Model = require "../model/model"
```

Comment:
It's good to stay consistent rather than make unrelated changes. However, at the very end of this sequence of changes, we should switch to plural form. I presume it should be added as a task to the meta-issue (@bryevdv, your comment would be appreciated).